scm-ui/ui-webapp/src/repos/branches/components/DefaultBranchTag.tsx

```diff
@@ -30,17 +30,16 @@ type Props = WithTranslation & {
   defaultBranch?: boolean;
 };
 
-const LeftMarginTag = styled(Tag)`
+const InheritVerticalAlignTag = styled(Tag)`
   vertical-align: inherit;
```

Comment:
Why did you remove the vertical-align in a different component but not here ? The default value is `baseline`, does it differ here ?

Reply:
Good hint. From my point of view, this also makes little sense.